src/Microsoft.Health.Fhir.Shared.Api/Features/Filters/OperationOutcomeExceptionFilterAttribute.cs

```diff
@@ -79,6 +80,19 @@ public override void OnActionExecuted(ActionExecutedContext context)
                         operationOutcomeResult.StatusCode = HttpStatusCode.ServiceUnavailable;
                         break;
                     case ResourceNotValidException _:
+                        if (context.ActionDescriptor is ControllerActionDescriptor)
```

Comment:
I believe this can be simplified to `if (context.ActionDescriptor is ControllerActionDescriptor controllerDescriptor)`

Reply:
Changed